packages/mobile/src/components/core/FlatList.tsx

```diff
@@ -15,13 +12,20 @@ import { CollapsibleTabNavigatorContext } from '../top-tab-bar'
 import { PlayBarChin } from './PlayBarChin'
 import { PullToRefresh, useOverflowHandlers } from './PullToRefresh'
 
-type FlatListProps = RNFlatListProps<any>
+declare module 'react' {
```

Comment:
Should we put this somewhere higher in the hierarchy?

Reply:
When i did, it seemed to break normal react so i got scared, but can try again

Reply:
I thought declaring modules like this was global anyways, I could be wrong though. Might require the `global` keyword

Reply:
updated, great call